app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) == 3 and command_list[0] == "mv":
+        _, source_file, destination_file = command_list
+        if not os.path.isdir(destination_file):
+            if (os.path.dirname(destination_file)
+                    == os.path.dirname(source_file)):
+                os.rename(source_file, destination_file)
+            else:
```

Comment:
Your code will not work when destination is a dir:
`mv file1.txt first_dir/second_dir/third_dir/`